### PR TITLE
Added case insensitive mode

### PR DESCRIPTION
**Pull request by mrfu2**
Adds a CLI argument `-i` to search for addresses case insensitive.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -40,3 +40,8 @@ $ cargo run -- -help
 - [ ] Create a release build
 - [ ] Improve API adding more options
 - [ ] Add more tests
+- [X] Add commandline argument for case-insensitive option (`-i`)
+- [ ] Add commandline argument to keep going after finding an address (`-k`)
+- [ ] Add commandline argument for saving results to file (`-o results.txt`)
+- [ ] Add commandline argument for using a file as input (`-f addresses.txt`)
+- [ ] Add commandline argument for Bech32 `bc1q` addresses (`-b`)
```
Comment (ndelvalle) [inline README.md:47]: Are you planning on contributing to help me achieve this? 😇

File: README.md [hunk 2]
```diff
@@ -40,3 +40,8 @@ $ cargo run -- -help
 - [ ] Create a release build
 - [ ] Improve API adding more options
 - [ ] Add more tests
+- [X] Add commandline argument for case-insensitive option (`-i`)
+- [ ] Add commandline argument to keep going after finding an address (`-k`)
```
[the remark this answers is not in the document]
Comment (mrfu2) [reply README.md:44]: If you take a look at, for example, https://github.com/samr7/vanitygen this implementation just keeps spitting out all addresses that match the search parameter until the user stops the application. Particularly useful if you want to leave your machine overnight generating cool vanity addresses and then just pick the coolest one when you stop it.

File: README.md (hunk 1 above)
Comment (mrfu2) [reply README.md:47]: Definitely! I am still getting the hang of Rust. I have 4 years of professional enterprise software engineering experience, so enough experience to know I don't know s*** yet :D
But I'd love to help and learn together.
These ideas are definitely on my roadmap of you'll allow me.

File: README.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (ndelvalle) [reply README.md:44]: That is a cool idea. `vanitygen` has some awesome stuff we can try to mimic, I love that it provides feedback on its rate of progress and  how many keys it has checked.

File: README.md (hunk 1 above)
Comment (ndelvalle) [reply README.md:47]: That is great! Yeah, I'm trying to learn Rust too, that was the reason I created the project. For sure your contributions are more than welcome!

**Pull request merged**